yt_dlp/extractor/videocampus_sachsen.py

```diff
@@ -157,30 +197,33 @@ def _real_extract(self, url):
             'host', 'id', 'tmp_id', 'display_id', 'embed_id')
         webpage = self._download_webpage(url, video_id or tmp_id, fatal=False) or ''
 
-        if not video_id:
-            video_id = embed_id or self._html_search_regex(
-                rf'src="https?://{host}/media/embed.*(?:\?|&)key=([0-9a-f]+)&?',
-                webpage, 'video_id')
+        formats, subtitles = [], {}
+        title = description = thumbnail = None
+
+        metadata = self._search_json(r'var\s+options\s*=\s*', webpage, 'player options', video_id or tmp_id or embed_id, default=None, fatal=False)
```

Comment:
- use `display_id`, that's what it's there for
- whitespace unnecessary after the `=` in the pattern
- passing a `default` implies `fatal=False`
```suggestion
        metadata = self._search_json(r'var\s+options\s*=', webpage, 'player options', display_id, default=None)
```